.gitignore

```diff
@@ -1,4 +1,28 @@
-.Rproj.user
+# History files
 .Rhistory
+.Rapp.history
+
+# Session Data files
 .RData
-.Ruserdata
+
+# Example code in package build process
+*-Ex.R
+
+# RStudio files
+.Rproj.user/
+
+# OAuth2 token, see https://github.com/hadley/httr/releases/tag/v0.3
+.httr-oauth
+.Rproj.user
+
+#data
+data/
+
+#figures & output
+*.png
+.html
```

Comment:
this will only block the file ".html" - needs a *.

Reply:
Oki fixed!